### ISSUE TITLE
New BehaviorSpace variable for repetitions

### ISSUE THREAD

**Issue by cstaelin** [NetLogo/NetLogo#1801]
It would be useful when analyzing the results of BehaviorSpace runs with repetitions > 1 to have a variable similar to `behaviorspace-run-number` that gives the repetition number, something like `behaviorspace-repetition-number`.  It would make it easier to compare results across repetitions of the same set of variable values in (say) R.

**Comment by digitaldust** [comment]
yes, apart from "behaviorspace-run-number”, which is an id of a specific run, it would be nice to have:

behaviorspace-exp-number 1 
behaviorspace-rep-number 56

to indicate that this is the 56th repetition of the combination of parameters in experiment 1

Best,
Simone

**Comment by alan-isaac** [comment]
There is already [access to the experiment name](https://ccl.northwestern.edu/netlogo/docs/dictionary.html#behaviorspace-experiment-name).

**Comment by digitaldust** [comment]
That’s a different thing, which says nothing about the params combination nor the repetition number of a specific combination of params within an experiment

**Comment by cstaelin** [comment]
So, I'm going to get more greedy.  In analyzing the results of multiple repetitions over multiple combinations of parameters, it would be nice to be able to identify a run number with both a particular set of parameters and a repetition number for that set, e.g., set 3, repetition 2.  I dump the results of my runs into R and I've written a function in R that will add those variables to the results of each run, but if they could go into a future version of NetLogo, that would be nice.

Thansk,
Charles
